lib/manageiq/providers/openstack/legacy/openstack_handle/handle.rb

```diff
@@ -353,7 +353,7 @@ def tenant_accessible?(name)
     end
 
     def default_tenant_name
-      return @default_tenant_name ||= "admin" if tenant_accessible?("admin")
+      return @default_tenant_name ||= ("admin" if tenant_accessible?("admin"))
```

Comment:
drive by. are you sure this is not:

```suggestion
      (return @default_tenant_name ||= "admin") if tenant_accessible?("admin")
```

Reply:
Yes I am :)